orchestrator-salt/src/main/resources/salt-common/salt/monitoring/textfiles.sls

```diff
@@ -0,0 +1,41 @@
+{%- from 'monitoring/settings.sls' import monitoring with context %}
+
+{%- if monitoring.enabled and monitoring.nodeExporterExists %}
+
+/var/lib/node_exporter/files:
+  file.directory:
+    - name: /var/lib/node_exporter/files
+    - user: "root"
+    - group: "root"
+    - mode: 750
+    - failhard: True
+    - makedirs: True
+
+/var/lib/node_exporter/scripts:
+  file.directory:
+    - name: /var/lib/node_exporter/scripts
+    - user: "root"
+    - group: "root"
+    - mode: 750
+    - failhard: True
+    - makedirs: True
+
+/etc/cron.d/salt_key_check:
+  file.managed:
+    - name: /etc/cron.d/salt_key_check
+    - source: salt://monitoring/textfiles/cron/salt_key_check
+    - user: "root"
+    - group: "root"
+    - mode: 600
+    - onlyif: test -d /srv/salt
```

Comment:
i think this could be the fastest way. we can also check salt-master systemd service or only check existence of `/var/run/salt-master.pid` (maybe that could be better?). with grains we can match ipv4 addresses against grains['master'] addresses to make sure the actual minion is a master, i did not found any boolean flag to check that

or a check like this can be around the states instead of the onlyif function